### ISSUE TITLE
Issue to save any data like Partners, configuration etc.

### ISSUE THREAD

**Issue by gns-pkulkarni** [OpenAS2/OpenAs2App#329]
Hello Guys,
I'm new to this and I'm trying to set-up openAS2. I'm doing following (Same set-up on windows and amazon linux 2 )

- I have set-up openAS2 server and running it on http://127.0.0.1:10080(URL is different on amazon linux 2) and restapi URL is baseuri="http://127.0.0.1:4080" in config.xml
- I have downloaded this repository and installed all required dependencies via node and npm
- I've added .env in WebUI folder with VUE_APP_OPENAS2_SERVER_URL=http://localhost:10080
- I'm able see UI and able to login with default username and pwd. I can see default partners and partnerships (MyCompany, PartnerA and PartnerB)
- **Issue is** whenever I add/edit/delete a partner the change is shown on UI but partnerships.XML was unchanged. I can see following message on openAS2 server as well
![image](https://github.com/OpenAS2/OpenAs2App/assets/32671275/9701a720-4d28-4c29-9dff-d29a51bd89f5)
![image](https://github.com/OpenAS2/OpenAs2App/assets/32671275/67d47036-0426-489e-baba-90a01be74e41)

- But partnerships.XML was unchanged, so when I restart the openAS2 server all the changes are gone and again I see the default partners and connections.
- This is the same problem I'm facing on both the platforms I've mentioned.

Can anyone please help? TIA

**Comment by uhurusurfa** [comment]
There is a PR open concerning this issue at the moment: https://github.com/OpenAS2/OpenAs2App/pull/327

In theory once this is merged the GUI config will work.

**Comment by uhurusurfa** [comment]
Closing as is duplicate of #319

**Comment by gns-pkulkarni** [comment]
@uhurusurfa I can see that PR. My problem is below

- I'm not using server from this repository. (Cause have some issue to run with .jar)
- I've downloaded sourceforge.net
- I'm using this repository for UI
- When I did what you mentioned in comment https://github.com/OpenAS2/OpenAs2App/pull/327#issuecomment-1633250978 it works well so I think It's just an issue with how UI is calling backend
- So when I did gh pr checkout 327, I've updated code from PR but issue was not solved. 
- Can you help me to let me know will this work once PR is merged or do I need to do something else? As my situation is bit different (I mean do I need to change anything on server side code)? Please help

**Comment by gns-pkulkarni** [comment]
Hello Guys,

Anyone can please help? @uhurusurfa any updates on PR and my queries if possible?

**Comment by uhurusurfa** [comment]
I will relese a new version that fixes this issue by end of the weekend.
